vars/parseStageInfo.groovy

```diff
@@ -112,5 +112,32 @@ def call(Map config = [:]) {
     if (config['ftest_arg']) {
       result['ftest_arg'] = config['ftest_arg']
     }
+
+    if (env.STAGE_NAME.contains('run_test.sh')) {
+      result['junit_files'] = 'test_results/*.xml'
+      result['artifacts'] = ['run_test.sh/*', 'vm_test/**']
+      result['with_valgrind'] = 'disabled'
```

Comment:
It seems that we can just leave this key unset if valgrind is not desired and then not enable valgrind if the key is unset.

Reply:
Okay, I can do that

Reply:
This variable ends up as an environment variable in `daos/ci/unit/test_post_always.sh`, and if this is not set to something the CI gives an unbound variable error. I like that test_post_always.sh doesn't change much in this pr https://github.com/daos-stack/daos/pull/3023 so probably I'll put back the `disabled` keyword.
I could remove the `set -u` in test_post_always.sh, but I don't think that's correct. I don't know about other options, let me know your comments.